docs/kb/semgrep-appsec-platform/saml-google-workspace.md

```diff
@@ -5,7 +5,7 @@ tags:
   - SSO
 ---
 
-# SAML Single Sign-on with Google Workspace
+# SAML single sign-on with Google Workspace
```

Comment:
Actually, after I looked at all the others, I realized they all say "SAML SSO". 

Possible options:
* Change this to `SAML SSO` too
* Change all the others to `SAML single sign-on`
* Distinguish between setup guides, which use the full name, and error troubleshooting, which has the abbreviation

Reply:
Normally, I would change everything to SAML single-sign on, but I think SSO is common enough that it can stand on its own, so I opted for SAML SSO. It also lines up with the error messages, which I think is helpful